src/main/kotlin/org/rust/lang/core/psi/ext/RsQualifiedNamedElement.kt

```diff
@@ -441,6 +468,7 @@ data class RsQualifiedName private constructor(
 
     enum class ChildItemType : ItemType {
         VARIANT,
+        FIELD,
```

Comment:
Unfortunately, no. External links for struct fields and fields of enum variants contains different identifiers
For example, in this case
```rust
fn foo(x: std::raw::TraitObject) {
    let data = x.data;
                         //^
}
```
we should generate `https://doc.rust-lang.org/core/raw/struct.TraitObject.html#structfield.data` instead of `https://doc.rust-lang.org/core/raw/struct.TraitObject.html#field.data`

Reply:
Oh, forgot to mention that we use name of each variant in external link generation